tasks/install.yml

```
@@ -3,7 +3,7 @@
   package:
     name: unzip
     state: present
-    update_cache: True
+    update_cache: promtail_apt_update_cache
```

```suggestion
    update_cache: "{{ promtail_apt_update_cache }}"
```
Unfortunately we need to use template interpolation in order to pass variables from defaults to the task

---

Applied.